src/main/java/hudson/plugins/sshslaves/SSHLauncher.java

```diff
@@ -435,7 +435,11 @@ public SSHLauncher(String host, int port, String username, String password, Stri
     @Deprecated
     public SSHLauncher(String host, int port, String username, String password, String privatekey, String jvmOptions,
                                     String javaPath, JDKInstaller jdkInstaller, String prefixStartSlaveCmd, String suffixStartSlaveCmd) {
-        this.host = host;
+        if (host != null) {
```

Comment:
Use `hudson.Util.fixEmptyAndTrim()`. It will also convert blank lines to null, which is maybe required here